pkg/graph/schema.graphql

```diff
@@ -1356,6 +1416,7 @@ type Query {
   deployments(cedarSystemId: String!, deploymentType: String, state: String, status: String): [CedarDeployment!]!
   roles(cedarSystemId: String!, roleTypeID: String): [CedarRole!]!
   detailedCedarSystemInfo(cedarSystemId: String!): DetailedCedarSystem
```

Comment:
This query and the `DetailedCedarSystem` type can probably be removed in favor of the `CedarSystemDetails` type you introduced, which has all of the same info.